main.go

```diff
@@ -10,14 +11,17 @@ import (
 	"os"
 	"path"
 	"path/filepath"
+	"regexp"
 	"sort"
 	"strings"
 )
 
 var (
-	flagOut    = flag.String("o", "", "Output file, else stdout.")
-	flagPkg    = flag.String("pkg", "main", "Package.")
-	flagPrefix = flag.String("prefix", "", "Prefix to strip from filesnames.")
+	flagOut      = flag.String("o", "", "Output file, else stdout.")
+	flagPkg      = flag.String("pkg", "main", "Package.")
+	flagPrefix   = flag.String("prefix", "", "Prefix to strip from filesnames.")
+	flagCondense = flag.Bool("condense", false, "Should we condense the compressed data in the generated source.")
```

Comment:
Did not want to decide for others but I'll definitely only use the base64 version :-)
Want me to change this?